pages/settings.js

```diff
@@ -0,0 +1,380 @@
+                    <input defaultValue={dataBot.botToken} 
+                        type='text' 
+                        name='botToken' 
+                        placeholder="Paste bot token" 
+                        onChange={(e) => setBotToken(e.target.value)}/>
+                </span>
+                <span className={settingsStyle.inputSpan}>
+                    <p className = {settingsStyle.nameField} >Hello message:</p>
+                    <textarea defaultValue={languageProfile.helloMessage} 
+                    placeholder="Write hello message" 
+                    name='helloMessage'
+                    onChange={(e) => setHelloMessage(e.target.value)}></textarea>
+                </span>
+                <span className={settingsStyle.inputSpan}>
+                    <p className = {settingsStyle.nameField}>Language:</p>
+                    <select name='id'>
+                        <option value={1}>English</option>
+                    </select>
+                    <p className={symbolsStyle.symbolPlus}>􀅼</p>
+                    <p className={symbolsStyle.symbolDelete}>􀈑</p>
+                </span>
+                <div className={settingsStyle.extraSettings}>
+                    <div className={settingsStyle.emojiesDiv}>
+                        <h2>Emojies:</h2>
+                        <span className={settingsStyle.inputSpanEmojie}>
+                            <p className = {settingsStyle.nameField}>Like:</p>
+                            <input defaultValue={dataBot.emojies.like} name='like' type='text'
+                            onChange={(e) => setLike(e.target.value)}/>
+                        </span>
+                        <span className={settingsStyle.inputSpanEmojie}>
+                            <p className = {settingsStyle.nameField}>Dislike:</p>
+                            <input defaultValue={dataBot.emojies.disLike} name='disLike' type='text'
+                            onChange={(e) => setDisLike(e.target.value)}/>
+                        </span>
+                        <span className={settingsStyle.inputSpanEmojie}>
+                            <p className = {settingsStyle.nameField}>Like + Message:</p>
+                            <input defaultValue={dataBot.emojies.message} name='message' type='text'
+                            onChange={(e) => setMessage(e.target.value)}/>
+                        </span>
+                        <span className={settingsStyle.inputSpanEmojie}>
+                            <p className = {settingsStyle.nameField}>Report:</p>
+                            <input defaultValue={dataBot.emojies.report} name='report' type='text'
+                            onChange={(e) => setReport(e.target.value)}/>
+                        </span>
+                    </div>
+                    <div className={settingsStyle.buttonsDiv}>
+                        <h2>Buttons:</h2>
+                        <span className={settingsStyle.inputSpanButton}>
```

Comment:
I suggest moving your spans into a component on its own and reuse it here.

Reply:
Fixed